src/modules/blocks/event-organizer/container.js

```diff
@@ -99,10 +102,26 @@ const mergeProps = ( stateProps, dispatchProps, ownProps ) => {
 	};
 };
 
+const StatefulEventOrganizer = ( props ) => {
+	useEffect( () => {
+		// Manage our initial state for defaults.
+		const defaults = editorDefaults();
+		const { attributes: { organizer } } = props;
+
+		if ( ! organizer && defaults && defaults.organizer ) {
+			props.setAttributes( { organizer: defaults.organizer } );
+		}
+	}, [] )
```

Comment:
```suggestion
	}, [ props ] );
```

Reply:
@EugeneKyale don't think we want this to re-run. It just loads defaults on initial run if we have a default available and if there isn't an existing organizer already set (assuming we will always have the `organizer` prop set on first render). Right?